uml/backend-software-system/Output-audio-handler.puml

```diff
@@ -0,0 +1,9 @@
+@startuml
+package "Audio Handler"{
```

Comment:
The package name should be "Sound Manager"  Isn't?

Reply:
i cant update something that its already merged, so i oppened a new PR with this fixes, becasue its not charging new commits to this PR but it does to the branch related to the US

Reply:
oke